Modules/_curses_panel.c

```diff
@@ -690,6 +690,8 @@ _curses_panel_exec(PyObject *mod)
 
 static PyModuleDef_Slot _curses_slots[] = {
     {Py_mod_exec, _curses_panel_exec},
+    // XXX gh-103092: fix isolation.
+    //{Py_mod_multiple_interpreters, Py_MOD_PER_INTERPRETER_GIL_SUPPORTED},
```

Comment:
Should be `Py_MOD_MULTIPLE_INTERPRETERS_NOT_SUPPORTED`, no? (Since it defaults to `Py_MOD_MULTIPLE_INTERPRETERS_SUPPORTED` if not set.)

Reply:
Yeah, you're right.  Good catch.

Reply:
There's a couple of more in the module list (_elementtree IIRC, and others)

Reply:
I've fixed them, but left faulthandler alone.  Things would get very annoying with crashes until faulthandler's isolation is finished.